### DISCUSSION TITLE
How to install "homebrew" for all the users on the server( globally) on RHEL 7

### DISCUSSION THREAD

**Discussion by shivak3** [Homebrew/discussions#3418] [Linux] [answered]
Hi All,

I am wondering how to install **"Homebrew" for all the users on RHEL 7 Linux server to use it on MULTI-USER System.**

I am trying to install homebrew under the path **"/opt/homebrew"** to be accessed by all users on the system. But, it is giving folder permission errors and not working.

Please guide me. Thank you in advance.

**Comment by SMillerDev** [comment, marked as answer]
> If you need to run Homebrew in a multi-user environment, consider creating a separate user account specifically for use of Homebrew.

https://docs.brew.sh/FAQ#why-does-homebrew-say-sudo-is-bad

**Comment by carlocab** [comment]
Trying creating a `linuxbrew` user with the home directory `/home/linuxbrew`, and then run the installer as this user.
